### PR TITLE
chore: update declare and deploy success screen layout

### PR DESCRIPTION
**Pull request by bluecco**
### Issue / feature description

Update declare and deploy success screen UI

### Changes

- `FlowHeader` based on `ShieldHeader` as a common ui component 

### Checklist

- [x] Rebased to the last commit of the target branch (or merged)
- [x] Code self-reviewed
- [x] Code self-tested
- [x] Tests updated (if needed)
- [x] All tests are passing locally

Please keep your pull request as small as possible. If you need to make multiple changes, please create separate pull requests for each change. Create a draft pull request if you need early feedback. This will mark the pull request as a work in progress and prevent it from being reviewed or merged until you are ready.

Please move drafts to the ready for review (regular PR) when you are ready to start the review process and other developers will pick it up from there.

### CODE REVIEW

**Comment by bluecco** [comment]
image

image

File: packages/ui/src/components/FlowHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+export interface ShieldHeaderProps {
+  title: ReactNode
+  subtitle?: ReactNode
+  variant?: ShieldHeaderVariant
+  size?: "md" | "lg"
+  icon?: ChakraComponent<"svg">
+  isLoading?: boolean
+}
+
+const FlowHeader: FC<ShieldHeaderProps> = ({
```
Comment (simonheys) [inline packages/ui/src/components/FlowHeader.tsx:45]: Should rename all `ShieldHeader…` to `FlowHeader…` ?

**Review by diegodelrieu — APPROVED** [review]

File: packages/ui/src/components/FlowHeader.tsx (hunk 1 above)
Comment (bluecco) [reply packages/ui/src/components/FlowHeader.tsx:45]: I thought I've missed something 😓

Comment (bluecco) [reply packages/ui/src/components/FlowHeader.tsx]: @simonheys updated

File: packages/ui/src/components/FlowHeader.tsx [hunk 2]
```diff
@@ -31,23 +31,23 @@ const variants = {
...
 }
 
-export const ShieldHeader: FC<ShieldHeaderProps> = ({
+const FlowHeader: FC<FlowHeaderProps> = ({
   title,
   subtitle,
   variant: variantKey = "default",
   size = "md",
-  icon: Icon = ArgentShieldIcon,
+  icon: Icon = TickIcon,
```
Comment (simonheys) [inline packages/ui/src/components/FlowHeader.tsx:50]: This is going to change assumptions about the default icon on shield screens? I will check the screens if so - let me know

Comment (bluecco) [reply packages/ui/src/components/FlowHeader.tsx:50]: you're right, updating the code

**Pull request merged**